app/code/Magento/GiftMessageGraphQl/Model/Resolver/GiftMessageCustomerOrder.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\GiftMessageGraphQl\Model\Resolver;
+
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\Framework\GraphQl\Query\Resolver\ContextInterface;
+use Magento\Framework\GraphQl\Query\Resolver\Value;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\GiftMessage\Api\OrderRepositoryInterface;
+
+/**
+ * Class for getting GiftMessage from CustomerOrder
+ */
+class GiftMessageCustomerOrder implements ResolverInterface
```

Comment:
Currently, we have 2 different resolvers in the directory called `GiftMessage` and `GiftMessageCustomerOrder`. 
The naming is not 100% consistent. So I would propose moving resolvers to the separate directories, so we have:

- `Model/Resolver/Order/GiftMessage.php`
- `Model/Resolver/Cart/GiftMessage.php`
- `Model/Resolver/Order/Item/GiftMessage` (in the future)

The same applies to the tests.

Reply:
**Done** for all three PRs
Thanks